enhancements/baremetal/baremetal-operator-management.md

```diff
@@ -0,0 +1,166 @@
+---
+title: baremetal-operator-management
+authors:
+  - "@markmc"
+reviewers:
+  - "@abhinavdahiya"
+  - "@dhellmann"
+  - "@enxebre"
+  - "@eparis"
+  - "@hardys"
+  - "@sadasu"
+  - "@smarterclayton"
+  - "@stbenjam"
+approvers:
+  - TBD
+creation-date: 2020-02-13
+last-updated: 2020-02-13
+status: provisional
+see-also:
+  - https://github.com/openshift/enhancements/pull/200
+replaces:
+superseded-by:
+---
+
+# Managing the Bare Metal Operator
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+1. Should the Bare Metal Operator be managed by the Cluster Version
+   Operator or the Machine API Operator?
```

Comment:
I believe it should be managed by the machine api operator.  CVO does not manage "operators", it manages resources.  It does not do conditional logic for operator deployment.  That's the responsibility of second level operators, of which MAO is one.

Reply:
Ok, it seems we're buying the need to separate (with stable APIs) the machine API (and its providers) as a consumer of infrastructure APIs from any components implementing infrastructure APIs.

And that bare metal is the unique case of the cluster hosting components that implement infrastructure APIs, and we should this as a standalone "subsystem" with its own SLO.

Reply:
New revision captures the "new subsystem" proposal